internal/ramses/reader.go

```diff
@@ -0,0 +1,58 @@
+package ramses
+
+import (
+	"encoding/binary"
+	"io"
+
+	"github.com/pkg/errors"
+)
+
+// Package is complete Ramses package
+type Package struct {
+	Header       Ramses
+	SecureHeader Secure
+	Payload      []byte
+}
+
+//Packages reads Ramses packages from buffer
+func Packages(buf io.Reader, packageChannel chan<- Package, errorChannel chan<- error) {
+	defer close(packageChannel)
+	defer close(errorChannel)
+	var err error
+	for {
+		header := Ramses{}
+		err = header.Read(buf)
+		if err != nil {
+			if err == io.EOF {
+				break
+			}
+			errorChannel <- err
```

Comment:
do we have test of this error handling? And also, as discussed on the next, wouldn't it be much more useful if error included the filename/key of the rac file?